src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md

```diff
@@ -3,369 +3,191 @@ path: "/docs/writing-own-tests/setting-up-galasa-project"
+
+
+## About the tests
+
+The _TestAccount.java_ and the _TestPayee.java_ source files show how a Core Manager can be injected into your test class before any test methods being used. 
+
+The _TestAccountExtended.java_ and the _TestPayeeExtended.java_ source files show:
+
+- How to obtain the `run-id` identifier by which an instance of running the test can be known. This can be useful for naming artifacts, logging or otherwise.
+
+- How to read a text file resource which is embedded within the test OSGi bundle. This can be useful for getting data files for use in the testing of the application.
+
+- How logging can be performed to help debug issues in the test code itself.
+
+- How a file that is created by the test run can be captured together with other test results for later viewing.
+
+You can find the templates that generate these Java files in the <a href="https://github.com/galasa-dev/cli/tree/main/pkg/cmd/templates/projectCreate/parent-project/test-project/src/main/java" target="_blank">Galasa cli repository</a>.
+
+
+## Building the example projects by using Maven
+
+Complete the following steps to build the example projects:
+
+1. On the command line, navigate into the top level parent folder. In our example the command is ```cd dev.galasa.example.banking```
+1. Invoke Maven to build the OSGi bundles by running the ```mvn clean install``` command.
+
+The built artifacts are typically placed in the _~/.m2/repository_ folder.
+
+Note: The tool currently does not generate build files for use with Gradle. 
+
+
+## Additional notes on the key elements of pom.xml files
+
+The following sections provide a little more information about some of the elements that are found within the various pom.xml files.
+
+### The parent pom.xml file elements
+
+- The `<project>` and `<modelVersion>` elements are standard prologues to a `pom.xml` file.
+- The `<groupId>` is used to group related Maven projects in a Maven repository. It is recommended (but not enforced) that all projects in a test stream share the same `groupId`.
+- The `<artifactId>` must be unique for each Maven project under a `groupId`. To prevent confusion, you should make it unique across `groupId`s. The `groupId` and `artifactId` can nominally be anything you choose, but if you were to ever consider publishing the project on Maven Central, you would have to ensure that they were unique across Maven Central. Because of this, and to avoid future name collisions, it is conventional to use (reversed) company domain names, which leads to patterns like `dev.galasa.example.banking`.
+- The `<version>` in this project is set to `0.1.0-SNAPSHOT` so that Galasa knows it needs to look for the latest copy every time it runs a test in automation. For local test running, it will check for the latest copy the first time a test is run in a day. It is unlikely that you will need to update the version unless you are specifically interested in maintaining multiple versions of the same test project.
```

Comment:
> For local test running, it will check for the latest copy the first time a test is run in a day.

Don't think that is true. For local test running it always get the only version of the test which is in your .m2 folder. Just delete the sentence as it jarrs with my understanding ?

Reply:
have removed